config.ini

```diff
@@ -23,6 +23,11 @@
     "ref" : "external",
     "overthewire" : "sc16",
     "cpu" : "fc32",
+    "gpio_bank" : "RXA",
+    "atr_rx" : "0x0006",
```

Comment:
0x06 =    0b 0000 0110
0x18 =    0b 0001 1000
0x60 =    0b 0110 0000
0b180 =0b1 1000 0000
Looks good

Pins for each are directly across from eachother, J15 on LFRX: https://www.ettus.com/product/details/LFRX